### ISSUE TITLE
Missing lang attribute for <html> elements

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#8212]
### Issue Summary

This is a report from experimental automated tests which I have only partly confirmed.

There are a number of `<html>` elements in our templates which are missing a `lang` attribute to indicate the page’s language, as well as the `dir` attribute. Here is the full desired definition from our `skeleton.html`:

```jinja
{% load wagtailadmin_tags i18n %}
{% get_current_language as LANGUAGE_CODE %}
{% get_current_language_bidi as LANGUAGE_BIDI %}
<html lang="{{ LANGUAGE_CODE }}" dir="{% if LANGUAGE_BIDI %}rtl{% else %}ltr{% endif %}">
```

### Steps to Reproduce

Here are the flagged templates:

```
wagtail/admin/templates/wagtailadmin/notifications/base.html
wagtail/project_template/project_name/templates/500.html
wagtail/templates/wagtailcore/login.html
wagtail/templates/wagtailcore/page.html
wagtail/templates/wagtailcore/password_required.html
wagtail/test/testapp/templates/tests/base.html
```

**Comment by jamesray** [comment]
@thibaudcolas on this one!
